bin/pt-online-schema-change

```diff
@@ -8879,7 +8879,7 @@ sub main {
       # it's a hack, like ignoring a problem instead of fixing it somehow. We
       # should take a look at the things that get printed in a "normal"
       # non-quiet run, and "if !quiet" them, and then do some kind of Logger.pm
-      # or Messager.pm module for a future release.
+      # or Message.pm module for a future release.
```

Comment:
Why did you modify this? This is some non-existent module name. If you think this is a typo, better to call it a Messenger.pm